### PR TITLE
Rename repo

### PR DESCRIPTION
**Pull request by aknysh**
## What
* Rename repo

## Why
* To conform to community standard: https://www.terraform.io/docs/registry/modules/publish.html

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
 ## Module Usage
 
 ### Argument Reference
 
-Note: this module is intended for use with existing VPC and existing
-Internet Gateway.
-You should use [tf_vpc](https://github.com/cloudposse/tf_vpc) module if
-you plan to use new (separate) VPC.
+Note: this module is intended for use with existing VPC and existing Internet Gateway.
+You should use [tf_vpc](https://github.com/cloudposse/terraform-aws-vpc) module if you plan to use new (separate) VPC.
```
Comment (osterman) [inline README.md]: `tf_vpc` is wrong

Comment (aknysh) [reply README.md]: fixed

File: README.md [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
-You should use [tf_vpc](https://github.com/cloudposse/tf_vpc) module if
-you plan to use new (separate) VPC.
+Note: this module is intended for use with existing VPC and existing Internet Gateway.
+You should use [terraform-aws-vpc](https://github.com/cloudposse/terraform-aws-vpc) module if you plan to use a new (separate) VPC.
 
-```
+```hcl
 module "subnets" {
-  source = "git::https://github.com/cloudposse/tf_subnets.git?ref=master"
+  source = "git::https://github.com/cloudposse/erraform-aws-dynamic-subnets.git?ref=master"
```
Comment (comeanother) [inline README.md]: Typo

**Comment by comeanother** [comment]
`terrafrom plan`
screenshot_10

**Pull request merged**